metrics/report/report_dockerfile/Dockerfile

```diff
@@ -9,29 +9,28 @@
 #  - pandoc
 #  - The report generation R files and helper scripts
 
-# I would have liked to have used from base rocker/verse, but that
-# does not have all the tools we need, and it seems that apt install
-# does not simply work - texlive for instance did not install.
-#
-# So, build up our image by hand then...
-FROM ubuntu
+# Start with the base rocker tidyverse.
+# We would have used the 'verse' base, that already has some of the docs processing
+# installed, but I could not figure out how to add in the extra bits we needed to
+# the lite tex version is uses.
+FROM rocker/tidyverse
 
 # Version of the Dockerfile
-LABEL DOCKERFILE_VERSION="1.0"
+LABEL DOCKERFILE_VERSION="1.1"
 
 # Without this some of the package installs stop to try and ask questions...
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && \
-	apt-get install -y tzdata r-base pandoc texlive
+# Install the extra doc processing parts we need for our Rmarkdown PDF flow.
+RUN apt-get update
+RUN apt-get install -y texlive-latex-base
```

Comment:
Running each `RUN` command separately is very clear, but I think every `RUN` gets converted to an image layer, so it's more efficient to chain all the commands you need together using oodles of `&&` and `\`'s.

Reply:
It's true. The opposite side of that is that if one 'later' item changes, the whole layer has to be rebuilt.
When I'm developing the Dockerfiles, I do them as separate `RUN`s, as then generally changes take effect quicker. But, sure, this is normally, hopefully, a one-off thing, so I'll squash some of those together.

Reply:
and pushed...